scaling-example/ansible.yaml

```diff
@@ -0,0 +1,213 @@
+tosca_definitions_version: cloudify_dsl_1_3
+
+description: >
+  This blueprint creates an AWS infrastructure environment using Ansible.
+  This is an example of blueprint that supports scale workflow.
+  libselinux-python/libselinux-python3 on the manager VM is prerequisite.
+
+imports:
+  - https://cloudify.co/spec/cloudify/6.0.0/types.yaml
+  - plugin:cloudify-ansible-plugin
```

Comment:
Should we constrain Ansible?

Reply:
Not really, I'm not using your new features only the run operation.